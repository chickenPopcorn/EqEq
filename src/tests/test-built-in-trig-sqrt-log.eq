SomeContext = {
  a = { 42; }

}

SomeContext:find a {
  print("%0.2f\n", cos(a));
  print("%0.2f\n", sin(a));
  print("%0.2f\n", tan(a));
  print("%0.2f\n", sqrt(a));
  print("%0.2f\n", log(a));
}
