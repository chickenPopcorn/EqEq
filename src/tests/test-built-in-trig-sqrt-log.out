-0.40
-0.92
2.29
6.48
3.74
